odoo/modules/graph.py

```diff
@@ -59,9 +59,9 @@ def add_modules(self, cr, module_list, force=None):
         len_graph = len(self)
         for module in module_list:
             # This will raise an exception if no/unreadable descriptor file.
-            # NOTE The call to load_information_from_description_file is already
-            # done by db.initialize, so it is possible to not do it again here.
-            info = odoo.modules.module.load_information_from_description_file(module)
+            # NOTE The call to get_manifest is already done by db.initialize,
+            # so it is possible to not do it again here.
+            info = odoo.modules.module.get_manifest(module)
```

Comment:
The comment no longer applies, since we now use a cached version of the function.
You should remove it.